templates/boilerplate/tsconfig.json

```diff
@@ -8,8 +8,8 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "module": "esnext",
```

Comment:
```
tsconfig.json:2:3 - error TS5098: Option 'customConditions' can only be used when 'moduleResolution' is set to 'node16', 'nodenext', or 'bundler'.
```

With the current setup, I received the above error. Hence modified the module and moduleResolution to address this problem.